analytical/templatetags/woopra.py

```diff
@@ -71,6 +71,63 @@ def _get_settings(self, context):
             variables['idle_timeout'] = str(settings.WOOPRA_IDLE_TIMEOUT)
         except AttributeError:
             pass
+        try:
+            variables['cookie_name'] = str(settings.WOOPRA_COOKIE_NAME)
+        except AttributeError:
+            pass
+        try:
+            variables['cookie_domain'] = str(settings.WOOPRA_COOKIE_DOMAIN)
+        except AttributeError:
+            pass
+        try:
+            variables['cookie_path'] = str(settings.WOOPRA_COOKIE_PATH)
+        except AttributeError:
+            pass
+        try:
+            variables['cookie_expire'] = str(settings.WOOPRA_COOKIE_EXPIRE)
+        except AttributeError:
+            pass
+        try:
+            variables['ping'] = str(settings.WOOPRA_PING)
+        except AttributeError:
+            pass
+        try:
+            variables['ping_interval'] = str(settings.WOOPRA_PING_INTERVAL)
+        except AttributeError:
+            pass
+        try:
+            variables['download_tracking'] = str(settings.WOOPRA_DOWNLOAD_TRACKING)
+        except AttributeError:
+            pass
+        try:
+            variables['outgoing_tracking'] = str(settings.WOOPRA_OUTGOING_TRACKING)
+        except AttributeError:
+            pass
+        try:
+            variables['outgoing_ignore_subdomain'] = str(settings.WOOPRA_OUTGOING_IGNORE_SUBDOMAIN)
+        except AttributeError:
+            pass
+        try:
+            variables['download_pause'] = str(settings.WOOPRA_DOWNLOAD_PAUSE)
+        except AttributeError:
+            pass
+        try:
+            variables['outgoing_pause'] = str(settings.WOOPRA_OUTGOING_PAUSE)
+        except AttributeError:
+            pass
+        try:
+            variables['ignore_query_url'] = str(settings.WOOPRA_IGNORE_QUERY_URL)
+        except AttributeError:
+            pass
+        try:
+            variables['map_query_params'] = str(settings.WOOPRA_MAP_QUERY_PARAMS)
+        except AttributeError:
+            pass
+        try:
+            variables['hide_campaign'] = str(settings.WOOPRA_HIDE_CAMPAIGN)
+        except AttributeError:
+            pass
+            
```

Comment:
I can't comment on whether this PR makes sense, because I don't know and use Woopra. So, I focus on Python and documentation:

- The excessive `try`-`except` is not DRY and makes the code less readable. Would you mind changing the implementation into a dictionary and a loop, e.g.
``` python
    woopra_settings = {
        'idle_timeout': 'WOOPRA_IDLE_TIMEOUT',
        'cookie_name': 'WOOPRA_COOKIE_NAME',
        'cookie_domain': 'WOOPRA_COOKIE_DOMAIN',
        'cookie_path': 'WOOPRA_COOKIE_PATH',
        'cookie_expire': 'WOOPRA_COOKIE_EXPIRE',
        'ping': 'WOOPRA_PING',
        'ping_interval': 'WOOPRA_PING_INTERVAL',
        'download_tracking': 'WOOPRA_DOWNLOAD_TRACKING',
        'outgoing_tracking': 'WOOPRA_OUTGOING_TRACKING',
        'outgoing_ignore_subdomain': 'WOOPRA_OUTGOING_IGNORE_SUBDOMAIN',
        'download_pause': 'WOOPRA_DOWNLOAD_PAUSE',
        'outgoing_pause': 'WOOPRA_OUTGOING_PAUSE',
        'ignore_query_url': 'WOOPRA_IGNORE_QUERY_URL',
        'map_query_params': 'WOOPRA_MAP_QUERY_PARAMS',
        'hide_campaign': 'WOOPRA_HIDE_CAMPAIGN',
    }
    for key, name in woopra_settings:
        try:
            variables[key] = str(getattr(settings, name))
        except AttributeError:
            pass
```
- The settings the code reads are not documented in the [Woopra section of our docs](https://github.com/jcassee/django-analytical/blob/master/docs/services/woopra.rst#configuration). Would you mind adding them to the docs?